### PR TITLE
FM2-137: Added mappings for relationshipType

### PR DESCRIPTION
**Pull request by corneliouzbett**
## Description of what I changed

Added relationshipType translator

## Issue I worked on

see https://issues.openmrs.org/browse/FM2-137

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**.

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/fhir2/api/translators/RelationshipTranslator.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+package org.openmrs.module.fhir2.api.translators;
+
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.openmrs.RelationshipType;
+
+public interface RelationshipTranslator extends ToFhirTranslator<RelationshipType, CodeableConcept> {
+	
+	/**
+	 * Maps {@link org.openmrs.RelationshipType } to a {@link org.hl7.fhir.r4.model.CodeableConcept}
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/fhir2/api/translators/RelationshipTranslator.java:18]: There's an extra space after RelationshipType. Normally a non-issue, but this does affect the formatting of the Javadoc.

**Comment by ibacher** [comment]
My one question about this: Relationships in OpenMRS are bi-directional. So x IS CHILD OF y implies y IS PARENT OF x. Does this capture this relationship bi-directionality?

**Comment by corneliouzbett** [comment]
> My one question about this: Relationships in OpenMRS are bi-directional. So x IS CHILD OF y implies y IS PARENT OF x. Does this capture this relationship bi-directionality?

No, it doesn't.
My challenge here is excluding relationships that involve a person who is involved in health care (e.g Doctor, nurse). RelatedPerson simply is a person related to the patient. 

Another thing is what should we do with **`Doctor-Patient`** relationships since It's not under `RelatedPerson` scope

**Comment by ibacher** [comment]
Doctor - Patient, Nurse - Patient, etc. are represented by the CareTeam FHIR resource.

I think the how we split this out is something we should ask on Talk, because if it’s just based on a string, that’s going to be hard to sort automatically.

**Comment by corneliouzbett** [comment]
> 2 yrs ->> 💀

**Pull request closed without merging**